Jasmine/Grid/ViewControllers/GridGameViewControllerDelegate.swift

```diff
@@ -5,9 +5,6 @@ import Foundation
 protocol GridGameViewControllerDelegate: BaseGameViewControllerDelegate {
 
     /// Update the grid data stored in the Grid Game View Controller with a new dataset.
-    ///
-    /// Note that this does *not* reload all the tiles displayed on the View Controller.
-    /// To do so, call a variant of `redisplayTiles`.
```

Comment:
should still be stated to avoid confusion.
perhaps use `- postconditions:` instead.